.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,27 @@
+# Avoids introducing vulnerable dependencies to the main branch.
+
+name: Dependency review
+on:
+  pull_request:
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # To write dependency submission
+      contents: write
+      # To write comment on PR
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v5
+
+      # Required because Dependabot Alerts doesn't natively support Scala
+      - uses: guardian/setup-scala@v1
+      - uses: scalacenter/sbt-dependency-submission@64084844d2b0a9b6c3765f33acde2fbe3f5ae7d3 # v3.1.0
```

Comment:
Does the dependency submission workflow need to run here too?

Reply:
Unfortunately to compare Scala deps it does because the branch dependency graph to compare with the main dependency graph isn't created automatically